### DISCUSSION TITLE
Error: Timed out waiting for language servers to shutdown

### DISCUSSION THREAD

**Discussion by crabdancing** [helix-editor/helix#8900] [General]
This is the output I get _every time_ I open a TOML file. Ever. It also hangs for over a second. Which isn't too big a deal, until I'm doing something complicated that involves opening and closing Helix a lot. I am using taplo 0.8.0, and Helix 23.10. 

Is there a way to configure Helix to just force kill a language server or language servers _in general_? I am really fine with the slight risk that _maybe_ some badly designed language server will write to its own config file, and thus corrupt it, or something.

**Comment by dead10ck** [comment]
This was an [upstream issue](https://github.com/tamasfe/taplo/pull/354) in taplo that was fixed. Is your taplo up to date?

**Comment by crabdancing** [comment]
Thank you for pointing that out. I'll be sure to bump the version and see what happens. But regardless of what happens, it seems like this workaround should be available for any misbehaving LSP on Helix's side, no?

**Reply by dead10ck** [reply]
Oh it looks like there is a PR open that got forgotten about.

https://github.com/helix-editor/helix/pull/5266
